packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -170,7 +180,7 @@ export default class WebpackConfigGenerator {
     parentPoint: WebpackPluginEntryPoint,
     entryPoint: WebpackPreloadEntryPoint,
   ) {
-    const rendererConfig = this.resolveConfig(this.pluginConfig.renderer.config);
+    const rendererConfig = await this.resolveConfig(this.pluginConfig.renderer.config);
     const prefixedEntries = entryPoint.prefixedEntries || [];
 
     return webpackMerge({
```

Comment:
@MOZGIII -- According to their docs, `webpack-merge` supports configuration objects and functions both. Why wasn't the function behaviour working as is?

Reply:
Aha. I answered my own question. They're lying about it. Sub-functions may be supported, but root functions aren't.